.github/ISSUE_TEMPLATE.md

```diff
@@ -1,24 +1,25 @@
 <!-- 
-Did you spot a typo or do you want to contribute to documentation? Instead of a GitHub issue, we encourage you to submit a pull-request (https://help.github.com/articles/creating-a-pull-request/) and we will review your changes!
+Thank you for your interest in contributing!
 
-For all other issues, please fill in the sections below.
+Before submitting an issue, check our existing issues and pull requests to see if someone else has reported your issue previously. 
 
-Check existing issues and pull-requests before submitting new ones, as a courtesy to the maintainers and making sure work isn't duplicated.
+If you know how to resolve the issue you're reporting, we highly encourage that you submit a pull request instead: https://help.github.com/articles/creating-a-pull-request/.
+
+If you have a troubleshooting issue, reach out to our community: https://github.com/Kong/kong/discussions.
 -->
 
-- `docs.konghq.com` url(s):
+### What part of the site needs attention?
+
+<!-- Link the part of the site that needs attention.  -->
 
 ### Summary
 
-SUMMARY_GOES_HERE
+<!-- 
+    Describe the issue in detail and, if applicable, tell us why this change is important.  
+        * Bad: "The ACL plugin documentation needs updating."
+        * Good: "Add the missing DELETE endpoint to ACL plugin documentation."
```

Comment:
```suggestion
        * Do: "Add the missing DELETE endpoint to ACL plugin documentation."
        * Don't: "The ACL plugin documentation needs updating."
```

Following the pattern we've got in the style guide already, plus it sounds more encouraging.